docs/develop/terrain/using-terrain-testnet.md

```diff
@@ -1,40 +1,37 @@
-# Use Terrain with the testnet
+# Use Terrain with testnet
 
 The [Pisco testnet](../../learn/glossary.md#testnet) is used for testing transactions on the Terra network.
 
 ## Prerequisites
 
-- [Install the Terra Station browser extension](../../learn/terra-station/download/terra-station-chrome.md)
+- Install the [Terra Station Chrome extension](../../learn/terra-station/download/terra-station-chrome.md).
 
-## Create a Pisco wallet
+## Create a test wallet
 
-[Create a new wallet](../../learn/terra-station/download/terra-station-chrome.md#create-a-wallet) using the Terra Station extension. It's recommended that you name this wallet "Pisco" or "testnet" so it's easy to remember.
+[Create a new wallet](../../learn/terra-station/download/terra-station-chrome.md#create-a-wallet) utilizing the Terra Station extension. It is recommended that you name this wallet something like "pisco" or "testnet" if you will use it only for testing just so it will be easier to remember.
```

Comment:
I think this can be simplified. 

Something like:

Open the Terra Station browser extension and [create a new wallet](../../learn/terra-station/download/terra-station-chrome.md#create-a-wallet). 

:::{tip}
Name your test wallet something that's easy to remember, such as "testnet" or "terrain". 
:::